### ISSUE TITLE
can we implement a page to show all online player ip

### ISSUE THREAD

**Issue by amazingfate** [spacemeowx2/slp-server-rust#22]
so that player can check the page and choose a usable ip address

**Comment by user890104** [comment]
Yes, GQL provides IP of both the server and the clients, check here for example web page client: https://github.com/initLab/games/blob/d56dc9cb5e99113f6d15b8466a62fb49035aee0b/public/index.html#L80
